modules/aws-frontend/variables.tf

```diff
@@ -56,3 +56,8 @@ variable "content_security_policy" {
     "worker-src"  = "blob:"
   }
 }
+
+variable "web_acl_id" {
+  type    = string
+  default = ""
```

Comment:
1. Fix missing description
2. Default value can be null, I think

Reply:
Making this values null makes it required, production will not use WAF so I think we should leave it as "" to make it optional, what do you think?

Reply:
I'm afraid that with `default=''`, cloudfront distribution resource will try to resolve empty string as id and throw an error. However, if you specify `default=null` then this variable will remain optional, but you won't be passing empty string to cloudfront distribution.

Reply:
When I put an empty string the parameter is ignored by the CloudFront module (it's not creating this var at all), with `default=null` the value is changed to required.